core/commands/multibase.go

```diff
@@ -0,0 +1,86 @@
+package commands
+
+import (
+	"bytes"
+	"io/ioutil"
+	"strings"
+
+	cmds "github.com/ipfs/go-ipfs-cmds"
+	"github.com/ipfs/go-ipfs/core/commands/cmdenv"
+	mbase "github.com/multiformats/go-multibase"
+)
+
+var MbaseCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Encode and decode files with multibase format",
+	},
+	Subcommands: map[string]*cmds.Command{
+		"encode": mbaseEncodeCmd,
+		"decode": mbaseDecodeCmd,
+	},
+	Extra: CreateCmdExtras(SetDoesNotUseRepo(true)),
+}
+
+const (
+	mbaseOptionName = "b"
+)
+
+var mbaseEncodeCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Encode file or stdin into multibase string",
+	},
+	Arguments: []cmds.Argument{
+		cmds.FileArg("file", true, false, "data to encode").EnableStdin(),
+	},
+	Options: []cmds.Option{
+		cmds.StringOption(mbaseOptionName, "multibase encoding").WithDefault("identity"),
```

Comment:
`identity` is not the best default, it will  cause some confusion:

```
$ echo lol | ipfs multibase encode
lol

$ echo lol | ipfs multibase decode
Error: selected encoding not supported
```

We should use something that works both ways out of the box, and has safe defaults.
I think base64url is both compact and safe (can be used in everywhere, including url paths) and makes a good default:

```suggestion
		cmds.StringOption(mbaseOptionName, "multibase encoding").WithDefault("base64url"),
```